### DISCUSSION TITLE
How to capture & replay video

### DISCUSSION THREAD

**Discussion by lehmannk** [fzyzcjy/flutter_convenient_test#390] [Q&A]
There are video replay buttons available for my tests but there are no videos to replay. How can I set this up (Windows)
<img width="989" alt="image" src="https://github.com/fzyzcjy/flutter_convenient_test/assets/36925534/b09f8ed6-81fa-44ba-aeaa-a80dd4375685">
The "Video" folder in the report folder is empty. The report bin file itself is present.

**Comment by fzyzcjy** [comment]
Yes, currently the vlc feature is disabled, because of https://github.com/fzyzcjy/flutter_convenient_test/issues/303.

However, if you want to use it, feel free to re-enable it locally, or make a PR using some video packages!
